### ISSUE TITLE
iteration number for fine-tune?

### ISSUE THREAD

**Issue by Tzy010822** [ShuzhaoXie/MesonGS#1]
Can you tell me the iteration number for fine-tune? 
From the code **gaussians.update_learning_rate(iteration+30000)**, it seems to mean that the learning rate is always the minimum value during the fine-tuning stage. Looking forward to your answer.

**Comment by ShuzhaoXie** [comment]
400-4000, more number of iteration maybe not help.
